test/intl402/NumberFormat/constructor-locales-hasproperty.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2018 Ujjwal Sharma. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-initializenumberformat
+description: >
+  Tests that HasProperty(O, Pk) is properly called within the constructor for
+  Intl.NumberFormat
+info: |
+  9.2.1 CanonicalizeLocaleList ( locales )
+
+  7.b. Let kPresent be ? HasProperty(O, Pk).
+---*/
+
+const locales = {
+  length: 8,
+  1: 'en-US',
+  3: 'de-DE',
+  5: 'en-IN',
+  7: 'en-GB'
+};
+
+const actualLookups = [];
+
+const handlers = {
+  has(obj, prop) {
+    actualLookups.push(prop);
+    return Reflect.has(...arguments);
+  }
+};
+
+const proxyLocales = new Proxy(locales, handlers);
+
+const nf = new Intl.NumberFormat(proxyLocales);
+
+assert.sameValue(actualLookups.length, locales.length);
+for (index in actualLookups) {
```

Comment:
`for (index in actualLookups) {` => `for (let index in actualLookups) {` (or it will fail in strict mode code)

Reply:
should be `const` to match the style in the rest of the file

Reply:
@rwaldron @mathiasbynens should I make another PR to change it, or let it be?